src/CoreWCF.CrankTests/CoreWCFPerfClient/Program.cs

```diff
@@ -0,0 +1,169 @@
+                BenchmarksEventSource.Register("corewcfperf/requests", Operations.Max, Operations.Sum, "Requests (" + test._paramPerfMeasurementDuration.TotalMilliseconds + " ms)", "Total number of requests", "n0");
+                BenchmarksEventSource.Register("corewcfperf/rps/max", Operations.Max, Operations.Sum, "Requests/sec (max)", "Max requests per second", "n0");
+
+                startTime = DateTime.Now;
+                ChannelFactory<ISayHello> factory = new ChannelFactory<ISayHello>(binding, new EndpointAddress(test._paramServiceUrl));
+                factory.Open();
+                var client = factory.CreateChannel();
+                ((IClientChannel)client).Open();
+                var stopwatchFirstReq = new Stopwatch();
+                stopwatchFirstReq.Start();
+                var result = client.HelloAsync("hello world").Result;
+
+                BenchmarksEventSource.Measure("firstrequest", stopwatchFirstReq.ElapsedMilliseconds);
+
+                while (DateTime.Now <= startTime.Add(measurementDurationPerTime))
+                {
+                    var rtnResult = await client.HelloAsync("hello world");
+                    Console.WriteLine(rtnResult);
+                    request++;
+                    requestTime = request;
+                    if (requestTime >= test._paramRequests & test._paramCloseAndOpen)
+                    {
+                        try
+                        {
+                            await Task.Factory.FromAsync(((IClientChannel)client).BeginClose, ((IClientChannel)client).EndClose, null);
+
+                        }
+                        catch (Exception ex)
+                        {
+                            Console.WriteLine($"Client Close failed : {ex}.");
+                            ((IClientChannel)client).Abort();
+                        }
+                        requestTime = 0;
+                    }
+
+                    if (((IClientChannel)client).State != CommunicationState.Opened)
+                    {
+                        client = factory.CreateChannel();
+                        await Task.Factory.FromAsync(((IClientChannel)client).BeginOpen, ((IClientChannel)client).EndOpen, null);
+                    }
+                }
+                factory.Close();
+                Console.WriteLine("ChannelFactory Closed.");
+
+                BenchmarksEventSource.Measure("corewcfperf/requests", request);
+                BenchmarksEventSource.Measure("corewcfperf/rps/max", request / test._paramPerfMeasurementDuration.TotalSeconds);
+            }
+        }
+
+        private bool ProcessRunOptions(string[] args)
+        {
+            foreach (string s in args)
+            {
+                Console.WriteLine(s);
+                string[] p = s.Split(new char[] { ':' }, count: 2);
+                if (p.Length != 2)
+                {
+                    continue;
+                }
```

Comment:
An argument which can't be parsed was passed on the command line. I don't think we should silently ignore it. I think passing an invalid command line argument should result in failing to run the test. Otherwise you risk a typo resulting in testing something different than intended.